cmake/std/PullRequestLinuxDriverMerge.py

```diff
@@ -167,3 +167,5 @@ def run():
         exit(0)
     else:
         exit(1)
+
```

Comment:
Extra whitespace

Reply:
Yes. I tend to prefer to add a couple extra empty lines to the bottom of scripts. It doesn't affect the results but gives me a small buffer at the end that's a little easier to see the end of code. Especially if I print out the source using `cat` that it gives a couple extra lines before the next prompt starts, etc.  Purely just a style thing that I tend to do.